ArduCopter/events.cpp

```diff
@@ -6,6 +6,9 @@
  */
 void Copter::failsafe_radio_on_event()
 {
+    // turn off artificial clipping on rc failsafe
+    ins.set_artificial_clipping(0);
```

Comment:
I am pretty sure we let the drone in trouble with other messing-up the controler function when we get a failsafe, but still nice to get this one !

Reply:
This particular change isn't going to go into master.  It's in one of the commits with "-- do not merge" in the title.  I'll remove it in the near future once we've completed all testing.